### PR TITLE
add script to compute different notions of fairness to the helper library

### PR DESCRIPTION
**Pull request by tschwedes**

### CODE REVIEW

File: src/helpers/helpers/fairness_measures.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    return ((scores >= 0.5) == labels).mean()
+
+def disparate_impact_p(scores, attr):
+    """
+    Computes disparate impact on probability level
+    
+    """
+    a_mask = attr == 1
+    di = np.abs(scores[~a_mask].mean() - scores[a_mask].mean())
+    return di
```
Comment (tcbegley) [inline src/helpers/helpers/fairness_measures.py]: May as well just return this directly

Comment (tcbegley) [reply src/helpers/helpers/fairness_measures.py]: Likewise below, there's a few more of these

File: src/helpers/helpers/fairness_measures.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                (scores[~a_mask] > bins[i]) & (scores[~a_mask] < bins[i + 1])
+            ).sum()
+            - (
+                (scores[a_mask & ~y_mask] > bins[i])
+                & (scores[a_mask & ~y_mask] < bins[i + 1])
+            ).sum()
+            / (
+                (scores[a_mask] > bins[i]) & (scores[a_mask] < bins[i + 1])
+            ).sum()
+        )
```
Comment (tcbegley) [inline src/helpers/helpers/fairness_measures.py]: Can we break this up and give each block a descriptive variable name? A bit hard to see what's going on here.

**Review by tcbegley — COMMENTED** [review]
Looks good. Left a few comments, main thing is the calibration metric. Since it's quite possible people will want to read our source code on these metrics it would be good to make that source code as readable as possible.

File: src/helpers/helpers/fairness_measures.py (hunk 1 above)
Comment (tschwedes) [reply src/helpers/helpers/fairness_measures.py]: Ok, agreed

File: src/helpers/helpers/fairness_measures.py (hunk 2 above)
Comment (tschwedes) [reply src/helpers/helpers/fairness_measures.py]: Yes, I'll make it more clear.

**Review by tcbegley — APPROVED** [review]

**Pull request merged**